iati/core/rulesets.py

```diff
@@ -232,13 +227,17 @@ def __init__(self, xpath_base, case):
         super(RuleAtLeastOne, self).__init__(xpath_base, case)
 
     def is_valid_for(self, dataset):
-        """Check `dataset.xml_tree` has at least one instance of a given case for an Element.
+        """Check dataset has at least one instance of a given case for an Element.
 
         Args:
-            dataset.xml_tree: an etree created from an XML dataset.
+            dataset (iati.core.Dataset): An IATI Dataset object.
```

Comment:
Out of the surrounding context, this isn't entirely clear. Would suggest something that is more standalone such as `The Dataset that is to have its validity checked against the Rule.` (but tweaked slightly to be less clunky)